src/CSI.IBTA.Administrator/Clients/AuthClient.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Text;
+using CSI.IBTA.Administrator.Interfaces;
+using Newtonsoft.Json;
+
+namespace CSI.IBTA.Administrator.Clients
+{
+    public class AuthClient : IAuthClient
+    {
+        private readonly HttpClient _httpClient;
+        private readonly ILogger<AuthClient> _logger;
+
+        public AuthClient(ILogger<AuthClient> logger, IConfiguration configuration)
+        {
+            _logger = logger;
+            _httpClient = new HttpClient();
+            var authServiceApiUrl = configuration.GetValue<string>("AuthServiceApiUrl");
+            if (string.IsNullOrEmpty(authServiceApiUrl))
+            {
+                _logger.LogError("AuthServiceApiUrl is missing in appsettings.json");
+                throw new InvalidOperationException("AuthServiceApiUrl is missing in appsettings.json");
+            }
+            _httpClient.BaseAddress = new Uri(authServiceApiUrl);
+        }
+
+        public async Task<HttpResponseMessage> PostAsync <T> (T dto, string apiEndpoint) where T : class
```

Comment:
```suggestion
       public async Task<AuthenticationResult> Authenticate(string username, string password);
```
Lack of abstraction. AuthClient should authenticate, not post requests. 
Update the method so is accepts credentials and returns some AuthenticationResult. Result will have bool property ( success / failure) and description string, describes why it failed. 
Change i-face accordignly.